plugin.json

```diff
@@ -4,6 +4,7 @@
     "description": "This plugin sends quarterly user satisfaction surveys to gather feedback and help improve Mattermost.",
     "homepage_url": "https://github.com/mattermost/mattermost-plugin-nps",
     "support_url": "https://github.com/mattermost/mattermost-plugin-nps/issues",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-nps/releases/tag/v1.1.0",
     "icon_path": "assets/icon.svg",
     "version": "1.0.4",
```

Comment:
It is normal/expected the tag of the release notes to point to a version different from the version?

Reply:
Only if you don't base your branch on the latest `master` :facepalm: